src/Elkuent/Builder.php

```diff
@@ -207,6 +215,56 @@ public function getFresh($columns = array())
     }
 
     /**
+     * Specify how to aggregate results
+     * Structured as an Elasticsearch aggregation: https://www.elastic.co/guide/en/elasticsearch/reference/1.5/search-aggregations.html
+     * 
+     * @param  array $aggregation PHP array representation of JSON aggregation object
+     * @return void
+     */
+    public function aggregateBy($aggregation) 
```

Comment:
can we make this `aggregateRaw` to match `whereRaw`?  I know we also have `getRawAggregation` but they signify raw input as well as raw output